packages/server/api/controllers/employeesController.js

```diff
@@ -0,0 +1,14 @@
+const knex = require('../../config/db');
+
+const fetchExistingMembers = async (req, res) => {
+  try {
+    const existingMembers = await knex('Employees').select('*');
+    res.status(200).json({ existingMembers });
```

Comment:
This is just a small thing, but I will replace `existingMembers` with `employees`.

Reply:
Thank you . It is a good suggestion.